tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -413,6 +413,7 @@ def load_rosparam(self, context, ros_config, cmd, param, file_, text, verbose=Tr
                 # #3162: if there is no YAML, load() will return an
                 # empty string.  We want an empty dictionary instead
                 # for our representation of empty.
+                
```

Comment:
Unrelated change. Should be be reverted.

Reply:
Reverted.